VCAC-A/ubuntu-18.04/analytics/dev/Dockerfile

```diff
@@ -525,6 +525,8 @@ RUN mkdir -p /home/build/usr/local/lib && \
     mkdir -p /home/build/usr/lib && \
     cp -r /usr/local/lib/* /home/build/usr/local/lib/ && \
     cp -rH /opt/intel/openvino /home/build/opt/intel/ && \
+    #fixing bug
+    cp /opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/libinference_engine_preproc.so /home/build/opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/libinference_engine_preproc.so && \
```

Comment:
Dev images already have libinference_engine_preproc.so, no need to duplicate it

Reply:
I agree. Pushed a patch